client/components/core/badge/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { privateApis } from '@wordpress/components';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import React from 'react';
+
+const CoreBadge = ( { children }: { children: React.ReactNode } ) => {
+	const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
```

Comment:
We may want to leave a comment for the future. If the badge component get stabilized, it will be removed from the private APIs, which will cause a build error.

We could leave a comment where we explain that, when the component gets stabilized, we should be able to import it directly from `@wordpress/components`

Reply:
Makes sense. Added it here: https://github.com/Automattic/wp-calypso/pull/99343/commits/d109811e7f5eacdf28d0487ca9f5173671b84697